yaml/landuse.yaml

```diff
@@ -756,6 +771,55 @@ filters:
       <<: *output_properties
       kind: beach
       tier: 5
+      # whitelist most common surface values for beaches. i'm not quite sure i'd
+      # call surface=grass a beach, but it's in the data..?
+      kind_detail:
+        case:
+          - when: {surface: [grass, gravel, pebbles, pebblestone, rocky, sand]}
```

Comment:
Looks like my favorite British English-ism https://wiki.openstreetmap.org/wiki/Tag:natural=shingle never factors in here (guess it would be rocky), looking at TagInfo.